### ISSUE TITLE
Button order is flipped on Windows for Discard changes dialog

### ISSUE THREAD

**Issue by anaisbetts** [desktop/desktop#7186]
## Description

On Windows, it's "Ok Cancel", not "Cancel Ok", even if the default action here is Cancel. Other dialogs seem to be correct though, kudos!

![image](https://user-images.githubusercontent.com/1396/55290871-ef0a9400-53d8-11e9-802a-260c4b4c4ad6.png)

## Version

* GitHub Desktop: 1.6.5

* Operating system: Windows 10

**Comment by niik** [comment]
:wave: @paulcbetts, thanks for opening this! We've iterated over this so many times that I think it's time for a fresh take on this and an attempt at addressing it more holistically.

For context, we do try to swap the button order depending on platform but this is a destructive/dangerous dialog where we also need to ensure that the primary button is the "safe" button and that's throwing a wrench in the works here.

I'm gonna propose this for inclusion in the release after 1.7.0, I've got some ideas that I want to test.
